packages/eslint-plugin/.eslintrc.json

```diff
@@ -22,6 +22,27 @@
     {
       "files": ["*.js", "*.jsx"],
       "rules": {}
+    },
+    {
+      "files": ["./package.json"],
+      "parser": "jsonc-eslint-parser",
+      "rules": {
+        "@nx/dependency-checks": [
+          "error",
+          {
+            "ignoredDependencies": [
+              "nx",
+              "tslib",
+              "typescript",
+              "eslint",
+              // Test-only
+              "memfs",
```

Comment:
Put it in devDeps instead of ignoring it?

Reply:
`devDependencies` wouldn't really make sense in our case since we don't install them.

Reply:
This is gone now that we ignore spec files by default (and users can customize it via `ignoreFilePatterns`).